utils/kata-manager.sh

```diff
@@ -10,6 +10,15 @@ set -o nounset
 set -o pipefail
 set -o errtrace
 
+#Added code :- simple checker, just sudo would require a 
+# passwword if sudo is not used before a restart
+if sudo modprobe vhost_vsock && sudo modprobe vhost_net; then
+  echo "VSOCK sockets and are supported"
```

Comment:
I don't think we need a message for the success case so I'd drop this. It might then make sense to invert the logic so that if we couldn't load those modules, we fail.

Finally, it might be an idea to structure this as a `for` loop. It takes up more lines of code but is clearer and easier to update if we decide to add further kernel modules to the list. Something like:

```bash
local -a modules=()

modules+=(vhost_vsock)
modules+=(vhost_net)

local module

for module in ${modules[@]}
do
    sudo modprobe "$module" || die "Host kernel does not support VSOCK sockets"
done
```